nvflare/edge/examples/README.md

```diff
@@ -1,70 +1,84 @@
-# Running Edge Example with Hierarchical Clients
+# Running NVFlare Mobile Example
 
-Please follow these steps to run the Edge device emulator,
 
-## Provision
+## Setup the NVFlare System
 
-Use tree_prov.py to generate a hierarchical NVFlare system with 2 levels and 2 clients at each level,
+```commandline
+bash setup_nvflare.sh
+```
 
-     python nvflare/edge/tree_prov.py -r /tmp -p edge_example -d 1 -w 2
+For details, please refer to [setup NVFlare system for Edge](./setup_system.md)
 
-This will create a deployment with 2 clients, 4 leaf-clients, 2 relays, 1 server.
+This will create a deployment with 2 clients, 4 leaf-clients, 2 relays, and 1 server.
```

Comment:
shall we update our words to Server - Aggregator - Leaf? @yanchengnv @nvidianz

Reply:
I think either leaf or leaf-clients are ok. But definitely change clients to aggregators.